iceoryx2-ffi/cxx/include/iox2/config_creation_error.hpp

```diff
@@ -25,6 +25,9 @@ enum class ConfigCreationError : uint8_t {
     /// Parts of the config file could not be deserialized. Indicates some kind of syntax error.
     UnableToDeserializeContents
 };
+
+auto error_string(const iox2::ConfigCreationError& error) -> const char*;
```

Comment:
We should use here the `From` approach like it is done here: https://github.com/eclipse-iceoryx/iceoryx2/blob/main/iceoryx2-ffi/cxx/include/iox2/enum_translation.hpp

Since it is a conversion from enum `iox2::ConfigCreationError` to `const char*`.

This applies to all `error_string()` free functions.